### PR TITLE
SYS-4404 Added systemd service support for all values.

### PR DESCRIPTION
**Pull request by firemdkfighter**

### CODE REVIEW

File: templates/openvpn_service.j2 [hunk 1]
```diff
@@ -16,8 +16,8 @@
 {{ ''}}
 {{ '[Service]' }}
 {% for s in vars['openvpn_systemd_settings'] %}
-{% if s.restart_on_failure is defined and s.restart_on_failure == 'true' %}
-{{ 'Restart=on-failure' }}
+{% if s.restart is defined and s.restart in ['on-success', 'on-failure', 'on-abnormal', 'on-watchdog', 'on-abort', 'always'] %}
+{{ 'Restart=' ~ s.restart }}
```
Comment (k0ste) [inline templates/openvpn_service.j2]: Operator should be `+` for string

File: README.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ openvpn:
       root_crt: 'rootCA.crt'
       root_key: 'rootCA.key'
   systemd_settings:
-  - restart_on_failure: 'true'
+  - restart: 'always'
```
Comment (k0ste) [inline README.md:118]: Documentation should be updated accordingly changes

**Review by k0ste — CHANGES_REQUESTED** [review]

File: templates/openvpn_service.j2 (hunk 1 above)
Comment (firemdkfighter) [reply templates/openvpn_service.j2]: You have the same operator above and below. https://github.com/k0ste/ansible-role-openvpn-ldap/blob/master/templates/openvpn_service.j2#L12

File: README.md (hunk 2 above)
Comment (firemdkfighter) [reply README.md:118]: Updated.

File: templates/openvpn_service.j2 (hunk 1 above)
Comment (k0ste) [reply templates/openvpn_service.j2]: Right! Because is may be `int` and should be explicitly convert to `str` before concatenation with string

**Comment by k0ste** [comment]
Thanks

**Pull request merged**